selfdrive/car/hyundai/hyundaican.py

```diff
@@ -69,6 +69,14 @@ def create_clu11(packer, frame, clu11, button):
   return packer.make_can_msg("CLU11", 0, values)
 
 
+def create_cancel_command(packer):
+  tcs13_values = {
+    "ACC_EQUIP": 1,
+    "PBRAKE_ACT": 1,
```

Comment:
When setting all the bits in byte 6, it brought the cancellation time down to ~10 frames and I narrowed it down to these two. If you set either one, it becomes more inconsistent and can take up to 70 frames to cancel